app/main.py

```
@@ -1 +1,18 @@
-# write your code here
+def copy_file(command: str) -> None:
+    check_command = command.split()
+    if len(check_command) == 3:
+        use_command = check_command[0]
+        file_use = check_command[1]
+        file_to_copy = check_command[2]
+        if file_use == file_to_copy:
+            pass
+        if use_command == "cp":
+            try:
+                with (open(file_use) as file_in,
+                      open(file_to_copy, "w") as file_out):
+                    for line in file_in.readlines():
+                        file_out.write(line)
+
+            except Exception:
+                pass
```

Checklist requirement: 'If the command is invalid, print an error message.'
Currently, exceptions are silently ignored with 'pass'. You should print an error message when an exception occurs.